consutil/main.py

```diff
@@ -22,9 +22,25 @@ def consutil():
 
 # 'show' subcommand
 @consutil.command()
-def line():
-    """Show all /dev/ttyUSB lines"""
-    click.echo("show line")
+def show():
+    """Show all /dev/ttyUSB lines and their info"""
+    devices = getAllDevices()
+    busyDevices = getBusyDevices()
+
+    click.echo("{:<4}   {:^6}   {:^5}   {:^25}".format("Line", "Baud", "PID", "Start Time"))
```

Comment:
Suggest using the Python "tabulate" module for formatting tabular input so as to be consistent with other SONiC CLI output. Refer to the source code of the `show` command for examples.